requirements/README.md

```diff
@@ -59,27 +59,27 @@ When we attemed to upgrade past 1.0.0 the build process in GitHub failed on the
 #19 0.725 Traceback (most recent call last):
 #19 0.725   File \"/var/lib/awx/venv/awx/bin/awx-manage\", line 8, in <module>
 #19 0.726     sys.exit(manage())
-#19 0.726   File \"/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/__init__.py\", line 178, in manage
+#19 0.726   File \"/var/lib/awx/venv/awx/lib64/python3/site-packages/awx/__init__.py\", line 178, in manage
```

Comment:
I love that you want to update the docs! This is great. But, let's please make sure we didn't just search & replace on these strings. This ouptut could be different on 3.11. Make sure you actually ran it again & copypasta the output instead of SRing the text.

Reply:
todo